src/content/member_profile_data.json

```diff
@@ -0,0 +1,180 @@
+{
+    "data": [
+        {
+            "profile": "https://github.com/aayush-05",
```

Comment:
this data should be downloaded dynamically from GitHub. It's a list with will change over time, we don't want to update it every time.